### DISCUSSION TITLE
Difficulty Adjust's increment is incorrect when Extended Limits is on

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#16745] [Q&A] [answered]
When you turn on Extended Limits, It increases by 0.6 instead of the usual 0.5 values.

https://user-images.githubusercontent.com/64581009/152168839-6fe1b239-f4a3-4a6f-9ccf-6d432bdb85ef.mp4

**Comment by frenzibyte** [comment, marked as answer]
Tracking in #16746.
